packages/gatsby-theme-store/src/components/Shelf/index.tsx

```diff
@@ -16,7 +19,7 @@ export interface Props {
   pageSizes?: number[]
   title?: JSX.Element | string
   variant?: string
-  showArrows: Maybe<boolean>
+  showArrows?: Maybe<boolean>
```

Comment:
`Maybe` doesn't already cover `undefined`?

Reply:
Yes, but I think it makes the prop required but accepting the value `undefined`. I'll remove the Maybe

Reply:
Nice, I think in that case we can use `showArrows?: boolean | null` 🤔 I don't like this because we have Maybe typing, but since it doesn't work... 😢